src/utils/accounts.js

```diff
@@ -85,6 +85,10 @@ export const findFirstSmartAccount = (accounts: Accounts): ?Account => {
   return accounts.find(({ type }) => type === ACCOUNT_TYPES.SMART_WALLET);
 };
 
+export const findFirstEtherspotAccount = (accounts: Accounts): ?Account => {
+  return accounts.find(({ type }) => type === ACCOUNT_TYPES.ETHERSPOT_SMART_WALLET);
+};
```

Comment:
Could be shorthanded to:

```suggestion
export const findFirstEtherspotAccount = (accounts: Accounts): ?Account => return accounts.find(({ type }) => type === ACCOUNT_TYPES.ETHERSPOT_SMART_WALLET);
```

Maybe? Just a suggestion.

Reply:
Thanks! Added more improved solution 👍